### ISSUE TITLE
Feature Request : Offline recovery instructions

### ISSUE THREAD

**Issue by ModerateIdeas** [cyphar/paperback#14]
If someone is storing paper archives for decades, they may not have access to the paperback software when they are trying to recover the data (internet issues, etc)

1. Would it be feasible to include manual decoding steps as an optional print out (some combination of command line tools?)
2. Would it be possible to encode a small decoding program as part of the archive?

**Comment by cyphar** [comment]
Yeah this is on the roadmap (I just forgot to open an issue about it). I spoke a little bit about this [in the talk on Paperback at Linux.conf.au 2022](https://www.youtube.com/watch?v=GI9rKdM9rB8). It would be impractical to implement decoding using command-line tools but I do plan to have a printable PDF containing a textual description of the algorithm and a Python (or some other programming language) program which can just do the recovery.

**Comment by detrin** [comment]
I like this idea. It would be cool to add at least an algorithmic description of how the data are stored. I think that it is likely that with the current speed of AI in two decades we could create a whole rust implementation of such reader given the description is precise enough.
